Assets/Scripts/Gamemode/Racing/RacingObjective.cs

```diff
@@ -58,7 +58,7 @@ private void SetColor(Color evenColor, Color unevenColor)
             {
                 var materialBlock = new MaterialPropertyBlock();
                 renderers[i].GetPropertyBlock(materialBlock);
-                materialBlock.SetColor("_Color", i % 2 == 0 ? evenColor : unevenColor);
+                materialBlock.SetColor("_BaseColor", i % 2 == 0 ? evenColor : unevenColor);
```

Comment:
Following the merge of issue #113 the code that updates the colors of the objectives was broken, to fix it I changed the string reference.